qcodes/instrument/parameter.py

```diff
@@ -2187,8 +2188,8 @@ def snapshot_base(self, update: bool = False,
         meta_data['label'] = param.label  # type: ignore[attr-defined]
         meta_data['full_name'] = param.full_name  # type: ignore[attr-defined]
         meta_data['aggregator'] = repr(getattr(self, 'f', None))
-        for param in self.parameters:
```

Comment:
This is the usual reusing a variable name issue for something else. Param is used above for the combinedparamter